template/tsconfig.electron-build.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "lib": ["DOM", "DOM.Iterable", "ES5"],
+    "isolatedModules": true,
+    "esModuleInterop": true,
+    "jsx": "react-jsx",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "target": "ES3",
+    "strict": true,
+    "noEmit": false,
+    "allowJs": true,
+    "checkJs": true,
+    "skipLibCheck": true
+  },
+  "include": [
+    "desktop/main.ts"
+  ],
+  "exclude": ["desktop/build", "public/build", ".cache"]
+}
```

Comment:
Hi @itsMapleLeaf 
Given that electron bundles a full version of a relatively modern version of chrome, I believe its safe to make these changes, which you significantly reduce the bundle size and app speed

- change `target to -> `target: esnext`
   - docs: https://www.typescriptlang.org/tsconfig#target
- change `lib` to: `lib: ["DOM", "DOM.Iterable", "esnext"]`
    - this way we get modern types for the browser environment and node
    - docs: https://www.typescriptlang.org/tsconfig#lib

Reply:
I think the rationale is a good one, and I'm pretty much always in favor of targeting modern JS, even outside of "stable" environments like Electron. But [here](https://github.com/itsMapleLeaf/remix-electron/pull/11#discussion_r818947752), I suggested to use tsup instead, which uses esbuild under the hood, which should accomplish the same goal, and changing the TSConfig target doesn't affect the emit.

Changing the lib is fine though